defaults/main.yml

```diff
@@ -42,6 +42,8 @@ camac_django_secret: 'rqo23kq9j&clk@ae5f43jhmz&qyan=6j-0ip0xcc5bauq4@$d8'
 
 camac_password_salt: "ds5fsdFd763znsPO"
 
+camac_application_env: "local"
```

Comment:
What is `local`? The readme says 'may be "development", "testing" or, "production"' so either the readme or the default here needs fixing.

Also, what's going on with `camac_env` vs. the new `camac_application_env`? Can't you just use one of them?

Reply:
Weird, the diff should have been outdated when you reviewed this. `camac_application_env` was removed.